source/glossary.rst

```diff
@@ -8,6 +8,12 @@ Glossary of Terms
      Headphone cueing, or just cueing, is listening to the next track you would
      like to mix in your headphones. The audience will not hear what you are
      cueing in your headphones. Being able to cue is a crucial aspect to DJing.
+     The shortcut for headphone cueing in Deck1 is T while for Deck 2 the shortcut is Y.
+
+   shortcuts
```

Comment:
referencing @Be-ing 's hint: please link this term in the manual (where it make sense) with :term:`shortcuts <shortcuts>` or :term:`keyboardhortcuts <shortcuts>` (depending on the context)

Reply:
> referencing @Be-ing 's hint: please link this term in the manual (where it make sense) with :term:`shortcuts <shortcuts>` or :term:`keyboardhortcuts <shortcuts>` (depending on the context)

Can you please explain what you meant here? I don't really understand. Thank you!

Reply:
We use :term:"display name <term_in_gloassary>" to link terms to respective paragraph in the Glossary
For example 'controllers' in https://github.com/mixxxdj/manual/blob/manual-2.2.x/source/chapters/controlling_mixxx.rst#using-a-keyboard links to https://www.mixxx.org/manual/latest/en/glossary.html#term-controller
(in the above exapmle replace **"** with **`** which unfortunately is the markdown shortcut for 'code' in github comments)

Reply:
@ronso0 just put the code on a new line and indent by four spaces, then you can use

    `

inside code blocks.

Reply:
```suggestion
   shortcut
   keyboard shortcut
```

Let's use the singular form here.